.github/workflows/release.yml

```diff
@@ -0,0 +1,50 @@
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  publish:
+    permissions:
+      id-token: write # required for SonarSource/vault-action-wrapper
+    runs-on: ubuntu-latest
+    env:
+      RELEASE_TAG: ${{ github.event.release.tag_name }}
+      NPM_REPOSITORY: "sonarsource-npm-public"
+      SCOPE: "@sonarsource"
```

Comment:
lets wait to have a final verdict on `@sonarsource` vs `@sonar`

Reply:
`@sonarsource` it is!

What's SB?

Reply:
storybook